packages/query/src/yargs.ts

```diff
@@ -19,6 +19,11 @@ export function getYargsOption() {
       type: 'boolean',
       demandOption: false,
     },
+    'log-dir': {
```

Comment:
To avoid using a mandatory file.log as log name, I suggest use `log-path` to allow user specify name. You can extract the dir via `path.dirname(path.resolve(filepath))`
Then I suggest to add another flag 'log-rotate` to enable rotate. (default to false)

Reply:
I have updated this.